gui/resources.xml

```diff
@@ -1104,6 +1141,7 @@ Vocabulary
           <property name="icon-name">mypaint-info-symbolic</property>
           <signal name="activate" handler="show_online_help_cb"/>
         </object>
+        <accelerator key="F1" modifiers="GDK_SHIFT_MASK"/>
```

Comment:
The only problem with that is that is a global shortcut depending on the DE so it will interfere with a music player sadly.

I did find out that F6, F7, F8 are free though.